modules/apiserversource-kn.adoc

```diff
@@ -70,15 +83,25 @@ Conditions:
 
 .Verification steps
 
-You can verify that the Kubernetes events were sent into the Knative eventing system by looking at the message dumper function logs.
+You can verify that the Kubernetes events were sent to Knative by looking at the message dumper function logs.
+
+* View the message dumper function logs by entering the following commands:
```

Comment:
I think these two commands should be split up into two steps because you are first getting the pods then retrieving logs for a pod.